pkg/daemon/controller/controller.go

```diff
@@ -316,33 +321,83 @@ func (c *Controller) GetIPInstanceSynced() cache.InformerSynced {
 //
 // Restart of vxlan interface will also trigger subnet and ip instance reconcile loop.
 func (c *Controller) handleLocalNetworkDeviceEvent() error {
+	hostNetNs, err := netns.Get()
+	if err != nil {
+		return fmt.Errorf("get root netns failed: %v", err)
+	}
+
 	linkCh := make(chan netlink.LinkUpdate, LinkUpdateChainSize)
```

Comment:
If `Subscribe` is created in loop, then L329-L332 is duplicated.

Reply:
Yes I forgot this. Fixed.